bahmni-emr/build.gradle

```diff
@@ -58,20 +145,21 @@ ospackage {
     }
 
     from("${projectDir}/resources/") {
-        fileMode = 0644
+        fileMode = 0755
```

Comment:
@shilpa-iplit why are permissions changing here? any specific reason?

Reply:
Reverted the changes. Please check the latest commit.